src/deepsparse/server/main.py

```diff
@@ -213,8 +213,11 @@ def server_app_factory():
     initialize_aysnc(config.workers)
     _LOGGER.debug("loaded server config %s", config)
     _add_general_routes(app, config)
-
-    pipelines = [Pipeline.from_config(model_config) for model_config in config.models]
+    context = Context()
```

Comment:
is there any num_cores to pass in here?

Reply:
good catch, num_cores needs to be set here, @rahul-tuli let's make sure this gets supported

Reply:
Addressed